StripeConnect/StripeConnectTests/Internal/Webview/ConnectWebViewTests.swift

```diff
@@ -112,10 +120,11 @@ class ConnectWebViewTests: XCTestCase {
                                       windowFeatures: .init())
 
         XCTAssertNil(webView)
-        wait(for: [canOpenURLExpectation, openURLExpectation], timeout: 0.1)
+        wait(for: [canOpenURLExpectation, openURLExpectation], timeout: TestHelpers.defaultTimeout)
     }
 
-    func testJavascriptPopupHandling() {
+    /// Any navigation request with a non-nil target should not open a popup
+    func testNonPopupNavigations() {
```

Comment:
I think this was misnamed? The test doesn't currently do any JS related testing.